.circleci/config.yml

```diff
@@ -12,76 +12,183 @@ version: 2.1
 orbs:
   win: circleci/windows@2.2.0
```

Comment:
### Recommended Change from CircleCi
Incorporate node orb

### Conclusion: Not incorporating at this time
The node orb is reliant on package versions being generated using the package-lock.json. Since we rely on the package.json we won't be incorporating this for now. However, we can revisit at a later date if we change to package-locks.